src/pages/index.php

```diff
@@ -4,10 +4,21 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content=""IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Login</title>
     <link rel=""stylesheet" href="assets/css/style.css"/>
 </head>
 <body>
-    
+    <form action="login.php" method="post">
+        <h2>LOGIN</h2>
+        <?php if (isset($_GET['error'])) { ?>
+            <p class="error"><?php echo $_GET['error']; ?></p>
+        <?php } ?>
+        <label>Email</label>
+        <input type="text" name="email" placeholder="Email"><br>
+        <label>Password</label>
+        <input type="password" name="password" placeholder="Password"><br>
+
+        <button type="submit">Login</button>
+    </form>
 </body>
```

Comment:
Say for this class index.php theres html in here too.
Jw is this what we going for or keeping all seperate or is this just how it must work for login?
//Olan

Reply:
Yeah I tried to work around it but with index it just gets messy and Im not sure what the best way is. So for that for the moment it is mixed but for home for example I split them up and will try to do with most